internal/pkg/deploy/cloudformation/stack/env.go

```diff
@@ -47,6 +47,24 @@ const (
 	envOutputManagerRoleKey      = "EnvironmentManagerRoleARN"
 )
 
+// S3 bucket tag keys.
+const (
+	EnvS3BucketStackNameTagKey = "aws:cloudformation:stack-name"
+	EnvS3BucketLogicalIDTagKey = "aws:cloudformation:logical-id"
```

Comment:
These tag key names do not only apply to S3 Bucket, nor env. I think you should be able to find the same tags (with different values) on ECS services as well. They will be tagged with the service stack name, and the `Service` logical ID.

```suggestion
	StackNameTagKey = "aws:cloudformation:stack-name"
	StackLogicalIDTagKey = "aws:cloudformation:logical-id"
```